### PR TITLE
[SKIP CI] Documentation update for ESXCLI usage and minor command fixes

### LINKED ISSUE #1757: Documentation update for admin-cli.md to reflect usage of esxcli based commands.
User facing (admin-cli.md) documentation should reflect usage of esxcli rather than vmdkops_admin.py.
This is to be done for all commands.

### PR DESCRIPTION
**Pull request by pshahzeb**
1. Updates documentation in admin-cli.md to reflect usage of esxcli
2. Update readme installation instruction to restart hostd after vib installation
3. Minor fixes to handle values with spaces for flags like name, vmgroup, datastore, description etc.
4. Introducing volume shortls command to display limited info about volumes.

Fixes #1757

### CODE REVIEW

**Comment by pshahzeb** [comment]
@shaominchen 
Yes we can come back and change the namespace once we finalize. Till then guestvol.

For shortls, yes it does sound a bit wierd. esxcli suggests the command name should be a verb.
However, given the functionality of command, I couldn't comeup with a better name,  which indicates volume ls with limited info.

**Comment by shuklanirdesh82** [comment]
@pshahzeb don't we have to update all markdown files? e.g. https://github.com/vmware/docker-volume-vsphere/blob/ee3ef339cb0703492e6d02f96fb4cb8a242065a1/docs/features/tenancy.md

**Comment by pshahzeb** [comment]
@shaominchen @shuklanirdesh82 
I have updated tenancy.md and faq.md files as well. Not touching vmdkops-admin-cli-spec.md
since it was the original spec file for vmdkops_admin.
Please take a look

File: esx_service/cli/vmdkops_admin.xml [hunk 1]
```diff
@@ -85,9 +85,68 @@
                 <formatter>table</formatter>
                 <format-parameter name="fields:vdvs">Volume,Datastore,VMGroup,Capacity,Used,Filesystem,Policy,Disk Format,Attached-to,Access,Attach-as,Created By,Created Date</format-parameter>
             </format-parameters>
-            <execute>/usr/lib/vmware/vmdkops/bin/vmdkops_admin.py --output-format=xml volume ls $if{vmgroup, --vmgroup $val{vmgroup}}</execute>
+            <execute>/usr/lib/vmware/vmdkops/bin/vmdkops_admin.py --output-format=xml volume ls $if{vmgroup, --vmgroup '$val{vmgroup}'}</execute>
         </command>
-        <command path="storage.vdvs.volume.set">
+        <command path="storage.guestvol.volume.shortls">
```
Comment (pshahzeb) [inline esx_service/cli/vmdkops_admin.xml]: Yes I thought so.
But on the basis of a flag, -v or (-c as earlier) we cannot decide on which columns to show.
The number of columns to be displayed for a command output is fixed and specified in esxcli xml extension.
The only option here is to introduce a new command.

**Pull request merged**